web/backend/src/Server.ts

```diff
@@ -287,6 +287,60 @@ app.use('/api/evoting/*', (req, res, next) => {
   }
 });
 
+// https://stackoverflow.com/a/1349426
+function makeid(length: number) {
```

Comment:
Since we are already using `short-unique-id` on the frontend, maybe using it too on the web backend should be better.

Reply:
In fact this code is a hack for a demo that I will present next week. It will be removed afterward.

Reply:
(but this is a very valid comment otherwise!)